Samples/watchOS-Swift/watchOS-Swift.yml

```diff
@@ -0,0 +1,60 @@
+name: watchOS-Swift
+createIntermediateGroups: true
+generateEmptyDirectories: true
+configs:
+  Debug: debug
+  Test: debug
+  TestCI: debug
+  Release: release
+projectReferences:
+  Sentry:
+    path: ../../Sentry.xcodeproj
+fileGroups:
+  - ../Shared/Config
+  - watchOS-Swift.yml
+  - ../../Plans/iOS-SwiftUI_Base.xctestplan
+options:
+  bundleIdPrefix: io.sentry
+targets:
+  "watchOS-Swift WatchKit App":
+    type: "application.watchapp2"
+    platform: auto
+    sources:
+      - "watchOS-Swift WatchKit App"
+      - ../Shared/SampleAssets.xcassets
+    dependencies:
+      - target: "watchOS-Swift WatchKit Extension"
+    configFiles:
+      Debug: watchOS-Swift-App.xcconfig
+      Release: watchOS-Swift-App.xcconfig
+      Test: watchOS-Swift-App.xcconfig
+      TestCI: watchOS-Swift-App.xcconfig
+  "watchOS-Swift WatchKit Extension":
```

Comment:
Note that this doesn't add an iOS target for `watchOS-Swift` as was present in the original project. I couldn't find any reference to it and figured it was just dead code.

Reply:
Ok thank you for the note.